server/autolinkclient/client.go

```diff
@@ -131,3 +109,20 @@ func (c *Client) Get(autolinkName string) ([]autolink.Autolink, error) {
 
 	return response, nil
 }
+
+func (c *Client) call(url string, body []byte, queryParams url.Values) (*http.Response, error) {
+	req, err := http.NewRequest(http.MethodGet, url, bytes.NewReader(body))
```

Comment:
there were different methods called previously, is it correct now? shouldn't we also pass the method as an argument?